libs/sdk-ui-kit/src/Header/generateHeaderAccountMenuItems.ts

```diff
@@ -0,0 +1,45 @@
+// (C) 2007-2021 GoodData Corporation
```

Comment:
missing UTs for all three generators

Reply:
I added UTs for all three generators thanks